setup.cfg

```
@@ -13,9 +13,9 @@ classifiers =
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
 project_urls =
-    Source Code = https://github.com/akaihola/pytest_darker
-    Change Log = https://github.com/akaihola/pytest_darker/CHANGES.rst
-url = https://github.com/akaihola/pytest_darker
+    Source Code = https://github.com/akaihola/pytest-darker
+    Change Log = https://github.com/akaihola/pytest-darker/CHANGES.rst
```

Change Log link isn't valid

---

Indeed, it should be
```suggestion
    Change Log = https://github.com/akaihola/pytest-darker/blob/master/CHANGES.rst
```